internal/cmd/plugin/hibernate/cmd.go

```diff
@@ -111,6 +111,7 @@ func NewCmd() *cobra.Command {
 			"text",
 			"Output format. One of text, json, or yaml",
 		)
+	plugin.AddColorControlFlag(hibernateStatusCmd)
```

Comment:
Adding the `--color` flag only in the hibernate or status subcommands is problematic IMO,
as users may find that adding it in other subcommands like fencing or backups, results in an exception.